projects/ad2s1210_iio/app/ad2s1210_user_config.c

```diff
@@ -0,0 +1,65 @@
+/***************************************************************************//**
+ * @file    ad2s1210_user_config.c
+ * @brief   User configurations for AD2S1210 No-OS driver
+********************************************************************************
+* Copyright (c) 2023 Analog Devices, Inc.
+* Copyright (c) 2023 BayLibre, SAS.
+* All rights reserved.
+*
+* This software is proprietary to Analog Devices, Inc. and its licensors.
+* By using this software you agree to the terms of the associated
+* Analog Devices Software License Agreement.
+*******************************************************************************/
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include "ad2s1210_user_config.h"
+#include "app_config.h"
+
+/******************************************************************************/
+/********************* Macros and Constants Definition ************************/
+/******************************************************************************/
+
+/******************************************************************************/
+/******************** Variables and User Defined Data Types *******************/
+/******************************************************************************/
+
+/* AD2S1210 No-OS driver init parameters */
+struct ad2s1210_init_param ad2s1210_init_params = {
+	.spi_init = { /* TODO: move to spi_init_param */
+		.max_speed_hz = 2000000,
+		.mode = NO_OS_SPI_MODE_1,
+		.chip_select = SPI_CSB,
+		.platform_ops = &spi_ops,
+		.extra = &spi_extra_init_params
+	},
+	.gpio_a0 = {
+		.number = ARDUINO_UNO_D0,
+		.platform_ops = &mbed_gpio_ops,
+		.extra = NULL
+	},
+	.gpio_a1 = {
+		.number = ARDUINO_UNO_D1,
+		.platform_ops = &mbed_gpio_ops,
+		.extra = NULL
+	},
+	.gpio_res0 = {
+		.number = ARDUINO_UNO_D5,
+		.platform_ops = &mbed_gpio_ops,
+		.extra = NULL
+	},
+	.gpio_res1 = {
+		.number = ARDUINO_UNO_D6,
+		.platform_ops = &mbed_gpio_ops,
+		.extra = NULL
+	},
+	.gpio_sample = {
+		.number = ARDUINO_UNO_D4,
+		.platform_ops = &mbed_gpio_ops,
+		.extra = NULL
+	},
+	.resolution = AD2S1210_RES_16BIT,
```

Comment:
Does this part allow the resolution to be configured to something other than 16?
If yes, it would be a good idea to add a macro in the app_config.h for selecting the same.
The macro can be used to assign the value of ``.resolution`` here as well

Reply:
yes we can use 16, 14,12,10. ill add the macro